bug: yarn watch is using watch.js

Hi! yarn watch is using watch.js

`
yarn watch
`

node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/home/scaffold-eth-typescript-challenges/packages/hardhat-ts/scripts/watch.js'

--- a/packages/hardhat-ts/package.json
+++ b/packages/hardhat-ts/package.json
@@ -10,7 +10,7 @@
     "deploy": "hardhat deploy --export-all ../vite-app-ts/src/generated/contracts/hardhat_contracts.json",
     "export": "hardhat export --export-all ../vite-app-ts/src/generated/contracts/hardhat_contracts.json",
     "postdeploy": "hardhat run scripts/publish.ts",
-    "watch": "node scripts/watch.js",
+    "watch": "ts-node scripts/watch.ts",
     "accounts": "hardhat accounts",
     "balance": "hardhat balance",
     "send": "hardhat send",

